docs/journalist.rst

```diff
@@ -1,289 +1,295 @@
-Workstation <tails_guide>` before continuing.
+If any sources have uploaded documents or sent messages, they will be
+listed on the homepage by codename.
 
-After clicking on the SecureDrop ``Journalist Interface`` link, you can
-log in with your username, password, and two-factor authentication
-token, as shown in the first screenshot below.
+|Journalist Interface|
 
-|Journalist Interface Login|
+.. note:: Codenames that journalists see are different than the
+          codenames visible to sources.
 
-If any sources have uploaded documents or sent you message, they will be
-listed on the homepage by a codename. **Note: The codename the
-journalists see is different than the codename that sources see.**
+Click on a codename to see the dedicated page for that source. You
+will see all of the messages that they have written and documents that
+they have uploaded. If the name of a source is difficult to say or
+remember, you can rename a source using the **Change codename** button
+next to their current codename.
 
-|Journalist Interface|
+|Cycle source codename|
 
-Move Documents to the *Secure Viewing Station*
-----------------------------------------------
+If you want to reply to the source, write your message in the text
+field and click **Submit**.
 
-You will only be able to view the documents the source has sent you on
-the *Secure Viewing Station*. After clicking on an individual source
-you will see the page below with the messages that source has sent you.
-Click on a document or message name to save it, or select a number of
-documents and save them at once by clicking **Download Selected**.
+|Sent reply|
 
-| |Load external content|
+Once your reply has been successfully submitted, you will be returned
+to the source page and see a message confirming that the reply was
+stored. The source will see your reply the next time they log in with
+their unique codename. To minimize sensitive data retention, the
+source interface UI encourages the source to delete the reply after
+reading it. If you notice one or more replies disappear from the list
+of documents, you may infer that the source read and deleted them. You
+may also delete replies if you change your mind after sending them.
 
-A dialog box will appear asking if you want to **Open** or **Save** the file.
-Select **Save File**:
+Documents and messages are encrypted to the SecureDrop installation's
+GPG public key. In order to read the messages or look at the documents
+you will need to transfer them to the *Secure Viewing Station*.
 
-| |Download selected|
 
-In order to protect you from malware, the browser only allows you to
-download documents to a special sandbox folder, located at
-``/home/amnesia/Tor Browser``.
+Flagging a Message for Reply
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-|Download to sandbox folder|
+If the server experiences a large number of new sources signing up at
+once and is overloaded with submissions, you will need to flag sources
```

Comment:
Another reason to flag for reply, if a potential source submits something that looks very promising.

Reply:
@freddymartinez9 - note what you are describing is not what "flag for reply" does. Flag for reply is used by journalists to indicate which sources should have keys generated if there is a large number of new sources (as the documentation currently describes). This should be left as is. Note there is another feature, the starring of sources which can be used to indicate which sources are of particular interest.

Reply:
This whole section on flagging for reply seems overly-technical to me. Maybe let's just say that there could be several reasons to flag, e.g. a source looks promising?